<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BindKeys</class>
 <widget class="QDialog" name="BindKeys">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>424</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Key Binds</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="11" column="4">
    <widget class="QPushButton" name="Close">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="QLabel" name="label_E">
     <property name="text">
      <string>E</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>3</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>2</string>
     </property>
    </widget>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="RestoreDefaults">
     <property name="text">
      <string>Restore Defaults</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_1">
     <property name="text">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QLabel" name="label_0">
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>9</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <widget class="QLabel" name="label_C">
     <property name="text">
      <string>C</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>6</string>
     </property>
    </widget>
   </item>
   <item row="10" column="3">
    <widget class="QLabel" name="label_F">
     <property name="text">
      <string>F</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>4</string>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QLabel" name="label_B">
     <property name="text">
      <string>B</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>7</string>
     </property>
    </widget>
   </item>
   <item row="9" column="3">
    <widget class="QLabel" name="label_D">
     <property name="text">
      <string>D</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>5</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>8</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_1"/>
   </item>
   <item row="2" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_2"/>
   </item>
   <item row="4" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_3"/>
   </item>
   <item row="4" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_4"/>
   </item>
   <item row="5" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_5"/>
   </item>
   <item row="5" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_6"/>
   </item>
   <item row="6" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_7"/>
   </item>
   <item row="6" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_8"/>
   </item>
   <item row="7" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_9"/>
   </item>
   <item row="7" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_0"/>
   </item>
   <item row="8" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_A"/>
   </item>
   <item row="8" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_B"/>
   </item>
   <item row="9" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_C"/>
   </item>
   <item row="9" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_D"/>
   </item>
   <item row="10" column="2">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_E"/>
   </item>
   <item row="10" column="4">
    <widget class="QKeySequenceEdit" name="keySequenceEdit_F"/>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_A">
     <property name="text">
      <string>A</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
